src/components/Layout/Layout.tsx

```diff
@@ -4,31 +4,33 @@ import React from 'react'
 
 import { Header } from './Header/Header'
 import { MobileNavBar } from './Header/NavBar/MobileNavBar'
-import { SideNav } from './Header/SideNav'
 
-const widthProp = { base: 'full', md: 'calc(100% - 93px)', '2xl': 'calc(100% - 384px)' }
+const widthProp = { base: 'full', md: 'full', '2xl': 'full' }
 const paddingBottomProp = {
   base: 'calc(0 + env(safe-area-inset-bottom) + var(--safe-area-inset-bottom))',
   md: 0,
 }
+const paddingXProp = { base: 0, md: 8, lg: 12 }
 
-export const Layout: React.FC<ContainerProps> = ({ children, ...rest }) => {
+export type LayoutProps = {
+  fullWidth?: boolean
```

Comment:
We already have `containerProps`, we don't need a fullWidth prop but pass the props directly when needed (px and maxWidth)